deeppavlov/models/preprocessors/siamese_preprocessor.py

```diff
@@ -69,6 +71,7 @@ def __init__(self,
                  use_matrix: bool = True,
                  num_context_turns: int = 1,
                  num_ranking_samples: int = 1,
+                 add_raw_text: bool = False,
                  tokenizer: Component = None,
                  vocab: Estimator = "simple_vocab",
```

Comment:
```suggestion
                 vocab: Optional[Estimator] = None,
```
The same for `embedder`. And then replace
```python3
if type(self.sent_vocab) != str:
```
with 
```python3
if self.sent_vocab is not None:
```